src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,21 +1,30 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends defaultCountingOutRhymer {
-
-	public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
-	
-	@Override
-	public int countOut() {
-		while (!callCheck())
-			
-		temp.countIn(super.countOut());
-		
-		int ret = temp.countOut();
-		
-		while (!temp.callCheck())
-			
-		countIn(temp.countOut());
-		
-		return ret;
-	}
+/**
+ * Class FIFORhymer used in RhymersFactory
+ */
+public class FIFORhymer extends DefaultCountingOutRhymer {
+
+    /**
+     *  instance created for operation and greater transparency
+     */
+    private final DefaultCountingOutRhymer TEMP = new DefaultCountingOutRhymer();
```

Comment:
Name should be written lowercase: this object is not constant (only reference is)